packit_service/service/models.py

```diff
@@ -141,12 +143,14 @@ def create(
         chroots: List[str],
         identifier: int = None,
         save: bool = True,
+        trigger_time: str = None,
     ):
         b = cls()
         b.identifier = identifier  # generate new ID if None
         b.project = project
         b.owner = owner
         b.chroots = chroots
+        b.trigger_time = datetime.now().strftime("%d-%m-%Y %H:%M:%S")
```

Comment:
I'd probably store just the timestamp as `int`, i.e.
```python
b.trigger_time: int = int(time.time())
```
One can later create a string from the timestamp according to preferences, for example
```python
datetime.utcfromtimestamp(b.trigger_time).isoformat()
```

Reply:
I actually removed it just a couple minutes ago. I set it to datetime.utcnow in both postgres and redis, using postgre's datetime field. 

This can be reformatted easily too, though reformatting might be an issue in other languages.

So should I store the timestamp?

Reply:
i was using it like this, and it worked perfectly
:+1:

Reply:
> i was using it like this, and it worked perfectly
It works as intended if you only modify the redis model.py (the old commit). 

I then modified the sql alchemy models.py to do the same thing. There is a bug in there which causes the error and then builds aren't stored.

Reply:
I'm fine with storing datetime, no need to store timestamp.